site/docs/api/serialization.md

```diff
@@ -84,11 +77,49 @@ Strategy                                                   Number of Bytes   Ove
 java.io.Serializable                                                   162          523
 java.io.Externalizable                                                  87          234
 com.hazelcast.nio.serialization.Portable                               104          300
-com.hazelcast.nio.serialization.DataSerializable                        88          238
-com.hazelcast.nio.serialization.IdentifiedDataSerializable              35           34
 com.hazelcast.nio.serialization.StreamSerializer                        26            0
 ```
 
+## Sample implementation
+
+For best performance we recommend using 
+[com.hazelcast.nio.serialization.StreamSerializer](https://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/nio/serialization/StreamSerializer.html) or
+[com.hazelcast.nio.serialization.ByteArraySerializer](https://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/nio/serialization/ByteArraySerializer.html).
+Below you can find a sample implementation of `StreamSerializer` for 
+`Person` (mind the type id which should be unique across all stream serializers):
+```
+class PersonSerializer implements StreamSerializer<Person> {
+
+        private static final int TYPE_ID = 1;
+
+        @Override
+        public int getTypeId() {
+            return TYPE_ID;
+        }
+
+        @Override
+        public void write(ObjectDataOutput out, Person person) throws IOException {
+            out.writeUTF(person.firstName);
+            out.writeUTF(person.lastName);
+            out.writeInt(person.age);
+            out.writeFloat(person.height);
+        }
+
+        @Override
+        public Person read(ObjectDataInput in) throws IOException {
+            return new Person(in.readUTF(), in.readUTF(), in.readInt(), in.readFloat());
+        }
+    }
+```
+
+Then the serializer should be registered with Jet on cluster startup:
+```
+JetConfig config = new JetConfig();
+config.getHazelcastConfig().getSerializationConfig()
+    .addSerializerConfig(new SerializerConfig().setTypeClass(Person.class).setClass(PersonSerializer.class));
+JetInstance jet = Jet.newJetInstance(config);
```

Comment:
I think worth mentioning is SerializerHooks, and also how to mention it when using a standalone cluster